### DISCUSSION TITLE
Chinese Terminology and Notes on Translation

### DISCUSSION THREAD

**Discussion by innerlee** [open-mmlab/mmediting#1217] [Announcements]
# 英中名词对照表

English | 中文 | 注
-- | -- | --
super-resolution | 超分辨率
checkpoint | 模型权重文件
backbone | 主干网络
cls_head | 分类头
pipeline | 流水线
hook | 钩子
finetune | 微调
register | 注册器
tensor | 张量
benchmark | 基准测试
recognizer | 动作识别器
detector | 检测器
annotation | 标注
training schedule | 训练策略
scheduler | 调度器
schedule | 调整
ground-truth | ？| 候选：人工标注
trimap | 三元图
refiner | ? | 候选：精炼器

## 其它常用词

English | 中文 | 注
-- | -- | --
Note | 注/注意 | 当内容属于 info 级别，用“注”；当内容属于 warn 级别，用“注意”

See also 
* https://github.com/open-mmlab/mmaction2/discussions/708

# 翻译注意事项

1. 中文字符与英文字符要用空格隔开。
    - 原文：The predicted alpha matte will be save in tests/data/pred/GT05.png.
    - 错误：预测的alpha遮罩图会保存在`tests/data/pred/GT05.png`目录下。
    - 正确：预测的 alpha 遮罩图会保存在 `tests/data/pred/GT05.png` 目录下。
2. 中文要通顺流畅。所谓”信达雅“，在不改变技术内容前提下，我们追求”雅“。
    - 原文：You can use the following commands to test a video for restoration.
    - 生硬：你可以使用如下命令来测试一段视频序列用于超分辨率重建。
    - 正常：您可以使用下列命令来超分辨率重建一段视频序列。
    - -
    - 原文：If you build PyTorch from source, CUDA 9.0 is also compatible
    - 翻译腔：如果要从源码对 PyTorch 进行编译, CUDA 9.0 版本同样可以兼容
    - 正常：如果从源码编译 PyTorch, 则可以兼容 CUDA 9.0
    - -
    - Install PyTorch and torchvision following the [official instructions](https://pytorch.org/)
    - 根据 [官方文档](https://pytorch.org/) 进行 PyTorch 和 torchvision 的安装
    - 参考[官方文档](https://pytorch.org/)安装 PyTorch 和 torchvision
    - -
    - You need to run `pip uninstall mmcv` first if you have mmcv installed.
    - 如果 mmcv 已经被安装，用户需要使用 `pip uninstall mmcv` 命令进行卸载。
    - 如果之前安装过 mmcv，那么需要先使用 `pip uninstall mmcv` 命令卸载它。
    - 
3. 保持原文的空行与缩进。markdown 格式有一定的要求，如 header 后要加空行、二级 list 元素要缩进等。故翻译时请保持原来的空行及缩进。
    - 原文 <br/> ![image](https://user-images.githubusercontent.com/9464825/126446554-6d07a139-c9cf-42d1-818f-71a802418489.png) <br/>原文的渲染结果 <br/>
![image](https://user-images.githubusercontent.com/9464825/126447335-c33bdcdc-3567-412d-acb6-2670d6eead02.png)
    - 错误 <br/> 
![image](https://user-images.githubusercontent.com/9464825/126446892-b861e382-2a7c-4629-9d84-4fbeedd4ae1d.png)<br/>缩进错误的渲染结果 <br/>
![image](https://user-images.githubusercontent.com/9464825/126447533-a4ce8a5c-cb4a-47fc-bded-942812f98199.png)
4. 人称。减少 “你/你的” 的使用频率。必须指代时，用敬语 “您/您的” 或中立词 “用户”
    - 原文：If you use the annotation mode for the dataset, you first need to prepare a specific `txt` file.
    - 生硬：如果你想使用`标注模式`来处理数据集，你需要先准备一个 `txt` 格式的标注文件。
    - 正常：如果您想使用`标注模式`来处理数据集，需要先准备一个 `txt` 格式的标注文件。

**Comment by innerlee** [comment]
*content moved to the first post*
